internals/testing/msw-server.ts

```diff
@@ -233,6 +233,10 @@ const handlers = [
     res(ctx.status(200), ctx.json(incidentFixture))
   ),
 
+  rest.patch(API.STANDARD_TEXTS_DETAILS_ENDPOINT, (_req, res, ctx) =>
```

Comment:
you're using two ways of mocking now. The GET inside the test, and then PATCH and DEL inside this MSW Server. Might be good to choose one.